examples/0000-test-json-config-small-network.zndsl

```diff
@@ -0,0 +1,16 @@
+Description: Small Network test
+Network: ./0000-test-config-small-network.json
+Creds: config
+
+# well known functions
+alice: is up
```

Comment:
This should not be needed anymore, since we are checking the nodes in the spawn phase.